suite/runsuite_wrapper.pl

```diff
@@ -243,6 +243,7 @@
                 'code_api,thread_private|common.decode-stress' => 1, # i#1807
```

Comment:
Reminder to add details to the PR description. And s/issues:/Issue:/

https://dynamorio.org/page_code_reviews.html